packages/squiggle-lang/__tests__/Distributions/AlgebraicShapeCombination_test.res

```diff
@@ -0,0 +1,16 @@
+open Jest
+open TestHelpers
+
+describe("combine with discrete", () => {
```

Comment:
We haven't been great at this, but i think the describe blocks should include the key thing being tested. Like, ``AlgebraicShapeCombination with discrete shapes..."

Reply:
Changed to something more specific